clarity_epp/export/manual_pipetting.py

```diff
@@ -502,16 +518,16 @@ def samplesheet_mip_multiplex_pool(lims, process_id, output_file):
         ))
 
     for input_artifact in process.all_inputs(resolve=True):
+        concentration = None
         # Find last qc process for artifact
-        qc_process = sorted(
-            lims.get_processes(type=qc_process_types, inputartifactlimsid=input_artifact.id),
-            key=lambda process: int(process.id.split('-')[-1])
-        )[-1]
+        qc_processes = lims.get_processes(type=qc_process_types, inputartifactlimsid=input_artifact.id)
 
-        # Find concentration measurement
-        for qc_artifact in qc_process.outputs_per_input(input_artifact.id):
-            if qc_artifact.name == input_artifact.name:
-                concentration = float(qc_artifact.udf['Dx Concentratie fluorescentie (ng/ul)'])
+        if qc_processes:
+            qc_process = sorted(qc_processes, key=lambda process: int(process.id.split('-')[-1]))[-1]
```

Comment:
So i am splitting the process id on `-`. The first part is linked to process type, second part is an integer which increases in value. So for same process you get id's like: `1-1`, `1-2`, `1-3`. I would like to sort the processes on the second part, so i am selecting `-1` -> `process.id.split('-')[-1]`.

Then I would like to select the highest number / the last qc process (date is unfortunately not reliable in Clarity since it does not include time). That is what the last `[-1]` is doing, reversing the order would just mean I have to select the first item -> `[1]`.

Reply:
Right, I got confused with the `::-1` format.